src/Sulu/Bundle/MediaBundle/Resources/config/list-builder/Collection.xml

```diff
@@ -0,0 +1,20 @@
+<class xmlns="http://schemas.sulu.io/class/general"
+       xmlns:list="http://schemas.sulu.io/class/list"
+       xmlns:orm="http://schemas.sulu.io/class/doctrine">
+    <properties>
+        <property name="id" list:translation="public.id" display="no" list:type="integer">
```

Comment:
Didn't we say we rename that to `sulu_admin.id`? But this probably also affects other list builder xmls, so this could be fixed together with #2149